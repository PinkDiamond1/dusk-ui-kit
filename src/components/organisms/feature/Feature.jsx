/* @flow */
import * as React from 'react'
import BootstrapJumbotron from 'react-bootstrap/Jumbotron'

import './Feature.scss'

type Props = {
  className: string,
  children: React.Node,
}

const Feature = (props: Props): React.Element<*> => {
  const { className, children } = props
  const updatedClassName = `__duk-feature ${className}`
  return <BootstrapJumbotron className={updatedClassName}>{children}</BootstrapJumbotron>
}

Feature.defaultProps = {
  className: '',
}

export default Feature
