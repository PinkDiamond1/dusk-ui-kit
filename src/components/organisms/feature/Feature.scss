.__duk-feature {
  background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiBpZD0iTGF5ZXJfMSIgeD0iMHB4IiB5PSIwcHgiIHZpZXdCb3g9IjAgMCAxMTIwIDY0MSIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMTEyMCA2NDE7IiB4bWw6c3BhY2U9InByZXNlcnZlIj4KPHN0eWxlIHR5cGU9InRleHQvY3NzIj4KCS5zdDB7Y2xpcC1wYXRoOnVybCgjU1ZHSURfMl8pO30KCS5zdDF7ZmlsbDojRkZGRkZGO30KCS5zdDJ7ZmlsbDojQzRDNEM0O2ZpbHRlcjp1cmwoI0Fkb2JlX09wYWNpdHlNYXNrRmlsdGVyKTt9Cgkuc3Qze21hc2s6dXJsKCNtYXNrMF8xXyk7fQoJLnN0NHtvcGFjaXR5OjAuMzQ7Y2xpcC1wYXRoOnVybCgjU1ZHSURfNF8pO30KCS5zdDV7ZmlsbDojNDQ0NDQ0O2ZpbGwtb3BhY2l0eTowLjQxO30KCS5zdDZ7ZmlsbDojREREREREO2ZpbGwtb3BhY2l0eTowLjQxO30KCS5zdDd7ZmlsbC1vcGFjaXR5OjAuNDE7fQoJLnN0OHtmaWxsOiM2NjY2NjY7ZmlsbC1vcGFjaXR5OjAuNDE7fQoJLnN0OXtmaWxsOiNBQUFBQUE7ZmlsbC1vcGFjaXR5OjAuNDE7fQoJLnN0MTB7ZmlsbDojOTk5OTk5O2ZpbGwtb3BhY2l0eTowLjQxO30KCS5zdDExe2ZpbGw6I0ZGRkZGRjtmaWxsLW9wYWNpdHk6MC40MTt9Cgkuc3QxMntmaWxsOiMyMjIyMjI7ZmlsbC1vcGFjaXR5OjAuNDE7fQoJLnN0MTN7b3BhY2l0eTowLjg1O2NsaXAtcGF0aDp1cmwoI1NWR0lEXzJfKTtmaWxsLXJ1bGU6ZXZlbm9kZDtjbGlwLXJ1bGU6ZXZlbm9kZDtmaWxsOiMzQjIxOEM7fQo8L3N0eWxlPgo8Zz4KCTxkZWZzPgoJCTxyZWN0IGlkPSJTVkdJRF8xXyIgd2lkdGg9IjExMjAiIGhlaWdodD0iNjQxIj48L3JlY3Q+Cgk8L2RlZnM+Cgk8Y2xpcFBhdGggaWQ9IlNWR0lEXzJfIj4KCQk8dXNlIHhsaW5rOmhyZWY9IiNTVkdJRF8xXyIgc3R5bGU9Im92ZXJmbG93OnZpc2libGU7Ij48L3VzZT4KCTwvY2xpcFBhdGg+Cgk8ZyBjbGFzcz0ic3QwIj4KCQk8Zz4KCQkJPHJlY3QgY2xhc3M9InN0MSIgd2lkdGg9IjExMjAiIGhlaWdodD0iNjQxIj48L3JlY3Q+CgkJCTxkZWZzPgoJCQkJPGZpbHRlciBpZD0iQWRvYmVfT3BhY2l0eU1hc2tGaWx0ZXIiPgoJCQkJCTxmZUNvbG9yTWF0cml4IHR5cGU9Im1hdHJpeCIgdmFsdWVzPSIxIDAgMCAwIDAgIDAgMSAwIDAgMCAgMCAwIDEgMCAwICAwIDAgMCAxIDAiPjwvZmVDb2xvck1hdHJpeD4KCQkJCTwvZmlsdGVyPgoJCQk8L2RlZnM+CgkJCTxtYXNrIG1hc2tVbml0cz0idXNlclNwYWNlT25Vc2UiIGlkPSJtYXNrMF8xXyI+CgkJCQk8cmVjdCBjbGFzcz0ic3QyIiB3aWR0aD0iMTEyMCIgaGVpZ2h0PSI2NDEiPjwvcmVjdD4KCQkJPC9tYXNrPgoJCQk8ZyBjbGFzcz0ic3QzIj4KCQkJPC9nPgoJCQk8Zz4KCQkJCTxkZWZzPgoJCQkJCTxyZWN0IGlkPSJTVkdJRF8zXyIgeD0iLTIwNi4zIiB5PSItNDI2LjYiIHdpZHRoPSIxNTM3LjQiIGhlaWdodD0iMTI4MS4xIj48L3JlY3Q+CgkJCQk8L2RlZnM+CgkJCQk8Y2xpcFBhdGggaWQ9IlNWR0lEXzRfIj4KCQkJCQk8dXNlIHhsaW5rOmhyZWY9IiNTVkdJRF8zXyIgc3R5bGU9Im92ZXJmbG93OnZpc2libGU7Ij48L3VzZT4KCQkJCTwvY2xpcFBhdGg+CgkJCQk8ZyBjbGFzcz0ic3Q0Ij4KCQkJCQk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNMTMzMS4xLTQyNi42SC0yMDYuM1Y4NTQuNWgxNTM3LjRWLTQyNi42eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDUiIGQ9Ik0tNzguMi0yMTMuMUwtMjA2LjMsMC40SDQ5LjlMLTc4LjItMjEzLjF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NiIgZD0iTTQzNC4zLTIxMy4xTDMwNi4yLDAuNGgyNTYuMkw0MzQuMy0yMTMuMXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q3IiBkPSJNNjkwLjUtMjEzLjFMNTYyLjQsMC40aDI1Ni4yTDY5MC41LTIxMy4xeiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDUiIGQ9Ik05NDYuNy0yMTMuMUw4MTguNiwwLjRoMjU2LjJMOTQ2LjctMjEzLjF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NiIgZD0iTTEyMDIuOS0yMTMuMUwxMDc0LjgsMC40aDI1Ni4yTDEyMDIuOS0yMTMuMXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q2IiBkPSJNLTc4LjIsMjE0bC0xMjguMSwyMTMuNUg0OS45TC03OC4yLDIxNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q3IiBkPSJNLTc4LjIsMjE0TDQ5LjksMC40aC0yNTYuMkwtNzguMiwyMTR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0OCIgZD0iTTE3OCwyMTRMNDkuOSw0MjcuNWgyNTYuMkwxNzgsMjE0eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDkiIGQ9Ik0xNzgsMjE0TDMwNi4yLDAuNEg0OS45TDE3OCwyMTR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NiIgZD0iTTQzNC4zLDIxNEwzMDYuMiw0MjcuNWgyNTYuMkw0MzQuMywyMTR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTAiIGQ9Ik00MzQuMywyMTRMNTYyLjQsMC40SDMwNi4yTDQzNC4zLDIxNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3QxMCIgZD0iTTY5MC41LDIxNEw1NjIuNCw0MjcuNWgyNTYuMkw2OTAuNSwyMTR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTEiIGQ9Ik02OTAuNSwyMTRMODE4LjYsMC40SDU2Mi40TDY5MC41LDIxNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q3IiBkPSJNOTQ2LjcsMjE0TDgxOC42LDQyNy41aDI1Ni4yTDk0Ni43LDIxNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q2IiBkPSJNOTQ2LjcsMjE0TDEwNzQuOCwwLjRIODE4LjZMOTQ2LjcsMjE0eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDkiIGQ9Ik0xMjAyLjksMjE0bC0xMjguMSwyMTMuNWgyNTYuMkwxMjAyLjksMjE0eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDUiIGQ9Ik0xMjAyLjksMjE0TDEzMzEuMSwwLjRoLTI1Ni4yTDEyMDIuOSwyMTR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTIiIGQ9Ik0tNzguMiw2NDFsLTEyOC4xLDIxMy41SDQ5LjlMLTc4LjIsNjQxeiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDciIGQ9Ik0xNzgsNjQxTDQ5LjksODU0LjVoMjU2LjJMMTc4LDY0MXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q2IiBkPSJNMTc4LDY0MWwxMjguMS0yMTMuNUg0OS45TDE3OCw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NyIgZD0iTTQzNC4zLDY0MWwxMjguMS0yMTMuNUgzMDYuMkw0MzQuMyw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NyIgZD0iTTY5MC41LDY0MUw1NjIuNCw4NTQuNWgyNTYuMkw2OTAuNSw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NSIgZD0iTTY5MC41LDY0MWwxMjguMS0yMTMuNUg1NjIuNEw2OTAuNSw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0OSIgZD0iTTk0Ni43LDY0MUw4MTguNiw4NTQuNWgyNTYuMkw5NDYuNyw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0OCIgZD0iTTk0Ni43LDY0MWwxMjguMS0yMTMuNUg4MTguNkw5NDYuNyw2NDF6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTAiIGQ9Ik0xMjAyLjksNjQxbC0xMjguMSwyMTMuNWgyNTYuMkwxMjAyLjksNjQxeiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDExIiBkPSJNNDkuOSwwLjRMLTc4LjIsMjE0SDE3OEw0OS45LDAuNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q4IiBkPSJNNDkuOSwwLjRMMTc4LTIxMy4xSC03OC4yTDQ5LjksMC40eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDEyIiBkPSJNMzA2LjIsMC40TDE3OCwyMTRoMjU2LjJMMzA2LjIsMC40eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDExIiBkPSJNMzA2LjIsMC40bDEyOC4xLTIxMy41SDE3OEwzMDYuMiwwLjR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NSIgZD0iTTU2Mi40LDAuNEw0MzQuMywyMTRoMjU2LjJMNTYyLjQsMC40eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDEyIiBkPSJNNTYyLjQsMC40bDEyOC4xLTIxMy41SDQzNC4zTDU2Mi40LDAuNHoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q5IiBkPSJNODE4LjYsMC40TDY5MC41LDIxNGgyNTYuMkw4MTguNiwwLjR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0OCIgZD0iTTgxOC42LDAuNGwxMjguMS0yMTMuNUg2OTAuNUw4MTguNiwwLjR6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTEiIGQ9Ik0xMDc0LjgsMC40TDk0Ni43LDIxNGgyNTYuMkwxMDc0LjgsMC40eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDEwIiBkPSJNMTA3NC44LDAuNGwxMjguMS0yMTMuNUg5NDYuN0wxMDc0LjgsMC40eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDkiIGQ9Ik00OS45LDQyNy41TC03OC4yLDY0MUgxNzhMNDkuOSw0MjcuNXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q1IiBkPSJNNDkuOSw0MjcuNUwxNzgsMjE0SC03OC4yTDQ5LjksNDI3LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0NSIgZD0iTTMwNi4yLDQyNy41TDE3OCw2NDFoMjU2LjJMMzA2LjIsNDI3LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTAiIGQ9Ik0zMDYuMiw0MjcuNUw0MzQuMywyMTRIMTc4TDMwNi4yLDQyNy41eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDgiIGQ9Ik01NjIuNCw0MjcuNUw2OTAuNSwyMTRINDM0LjNMNTYyLjQsNDI3LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTIiIGQ9Ik04MTguNiw0MjcuNUw2OTAuNSw2NDFoMjU2LjJMODE4LjYsNDI3LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTEiIGQ9Ik0xMDc0LjgsNDI3LjVMOTQ2LjcsNjQxaDI1Ni4yTDEwNzQuOCw0MjcuNXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3QxMiIgZD0iTTEwNzQuOCw0MjcuNUwxMjAyLjksMjE0SDk0Ni43TDEwNzQuOCw0MjcuNXoiPjwvcGF0aD4KCQkJCQk8cGF0aCBjbGFzcz0ic3Q1IiBkPSJNNDkuOSw4NTQuNUwxNzgsNjQxSC03OC4yTDQ5LjksODU0LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTEiIGQ9Ik0zMDYuMiw4NTQuNUw0MzQuMyw2NDFIMTc4TDMwNi4yLDg1NC41eiI+PC9wYXRoPgoJCQkJCTxwYXRoIGNsYXNzPSJzdDkiIGQ9Ik01NjIuNCw4NTQuNUw2OTAuNSw2NDFINDM0LjNMNTYyLjQsODU0LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTEiIGQ9Ik04MTguNiw4NTQuNUw5NDYuNyw2NDFINjkwLjVMODE4LjYsODU0LjV6Ij48L3BhdGg+CgkJCQkJPHBhdGggY2xhc3M9InN0MTIiIGQ9Ik0xMDc0LjgsODU0LjVMMTIwMi45LDY0MUg5NDYuN0wxMDc0LjgsODU0LjV6Ij48L3BhdGg+CgkJCQk8L2c+CgkJCTwvZz4KCQk8L2c+Cgk8L2c+Cgk8cmVjdCBjbGFzcz0ic3QxMyIgd2lkdGg9IjExMjAiIGhlaWdodD0iNjQxIj48L3JlY3Q+CjwvZz4KPC9zdmc+');
  background-attachment: fixed;
  border-radius: 0;
}
