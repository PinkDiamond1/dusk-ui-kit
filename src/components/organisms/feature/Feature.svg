<?xml version="1.0" encoding="UTF-8"?>
<!-- Generator: Adobe Illustrator 23.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Layer_1" x="0px" y="0px" viewBox="0 0 1120 641" style="enable-background:new 0 0 1120 641;" xml:space="preserve">
<style type="text/css">
	.st0{clip-path:url(#SVGID_2_);}
	.st1{fill:#FFFFFF;}
	.st2{fill:#C4C4C4;filter:url(#Adobe_OpacityMaskFilter);}
	.st3{mask:url(#mask0_1_);}
	.st4{opacity:0.34;clip-path:url(#SVGID_4_);}
	.st5{fill:#444444;fill-opacity:0.41;}
	.st6{fill:#DDDDDD;fill-opacity:0.41;}
	.st7{fill-opacity:0.41;}
	.st8{fill:#666666;fill-opacity:0.41;}
	.st9{fill:#AAAAAA;fill-opacity:0.41;}
	.st10{fill:#999999;fill-opacity:0.41;}
	.st11{fill:#FFFFFF;fill-opacity:0.41;}
	.st12{fill:#222222;fill-opacity:0.41;}
	.st13{opacity:0.85;clip-path:url(#SVGID_2_);fill-rule:evenodd;clip-rule:evenodd;fill:#3B218C;}
</style>
<g>
	<defs>
		<rect id="SVGID_1_" width="1120" height="641"></rect>
	</defs>
	<clipPath id="SVGID_2_">
		<use xlink:href="#SVGID_1_" style="overflow:visible;"></use>
	</clipPath>
	<g class="st0">
		<g>
			<rect class="st1" width="1120" height="641"></rect>
			<defs>
				<filter id="Adobe_OpacityMaskFilter">
					<feColorMatrix type="matrix" values="1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 1 0"></feColorMatrix>
				</filter>
			</defs>
			<mask maskUnits="userSpaceOnUse" id="mask0_1_">
				<rect class="st2" width="1120" height="641"></rect>
			</mask>
			<g class="st3">
			</g>
			<g>
				<defs>
					<rect id="SVGID_3_" x="-206.3" y="-426.6" width="1537.4" height="1281.1"></rect>
				</defs>
				<clipPath id="SVGID_4_">
					<use xlink:href="#SVGID_3_" style="overflow:visible;"></use>
				</clipPath>
				<g class="st4">
					<path class="st1" d="M1331.1-426.6H-206.3V854.5h1537.4V-426.6z"></path>
					<path class="st5" d="M-78.2-213.1L-206.3,0.4H49.9L-78.2-213.1z"></path>
					<path class="st6" d="M434.3-213.1L306.2,0.4h256.2L434.3-213.1z"></path>
					<path class="st7" d="M690.5-213.1L562.4,0.4h256.2L690.5-213.1z"></path>
					<path class="st5" d="M946.7-213.1L818.6,0.4h256.2L946.7-213.1z"></path>
					<path class="st6" d="M1202.9-213.1L1074.8,0.4h256.2L1202.9-213.1z"></path>
					<path class="st6" d="M-78.2,214l-128.1,213.5H49.9L-78.2,214z"></path>
					<path class="st7" d="M-78.2,214L49.9,0.4h-256.2L-78.2,214z"></path>
					<path class="st8" d="M178,214L49.9,427.5h256.2L178,214z"></path>
					<path class="st9" d="M178,214L306.2,0.4H49.9L178,214z"></path>
					<path class="st6" d="M434.3,214L306.2,427.5h256.2L434.3,214z"></path>
					<path class="st10" d="M434.3,214L562.4,0.4H306.2L434.3,214z"></path>
					<path class="st10" d="M690.5,214L562.4,427.5h256.2L690.5,214z"></path>
					<path class="st11" d="M690.5,214L818.6,0.4H562.4L690.5,214z"></path>
					<path class="st7" d="M946.7,214L818.6,427.5h256.2L946.7,214z"></path>
					<path class="st6" d="M946.7,214L1074.8,0.4H818.6L946.7,214z"></path>
					<path class="st9" d="M1202.9,214l-128.1,213.5h256.2L1202.9,214z"></path>
					<path class="st5" d="M1202.9,214L1331.1,0.4h-256.2L1202.9,214z"></path>
					<path class="st12" d="M-78.2,641l-128.1,213.5H49.9L-78.2,641z"></path>
					<path class="st7" d="M178,641L49.9,854.5h256.2L178,641z"></path>
					<path class="st6" d="M178,641l128.1-213.5H49.9L178,641z"></path>
					<path class="st7" d="M434.3,641l128.1-213.5H306.2L434.3,641z"></path>
					<path class="st7" d="M690.5,641L562.4,854.5h256.2L690.5,641z"></path>
					<path class="st5" d="M690.5,641l128.1-213.5H562.4L690.5,641z"></path>
					<path class="st9" d="M946.7,641L818.6,854.5h256.2L946.7,641z"></path>
					<path class="st8" d="M946.7,641l128.1-213.5H818.6L946.7,641z"></path>
					<path class="st10" d="M1202.9,641l-128.1,213.5h256.2L1202.9,641z"></path>
					<path class="st11" d="M49.9,0.4L-78.2,214H178L49.9,0.4z"></path>
					<path class="st8" d="M49.9,0.4L178-213.1H-78.2L49.9,0.4z"></path>
					<path class="st12" d="M306.2,0.4L178,214h256.2L306.2,0.4z"></path>
					<path class="st11" d="M306.2,0.4l128.1-213.5H178L306.2,0.4z"></path>
					<path class="st5" d="M562.4,0.4L434.3,214h256.2L562.4,0.4z"></path>
					<path class="st12" d="M562.4,0.4l128.1-213.5H434.3L562.4,0.4z"></path>
					<path class="st9" d="M818.6,0.4L690.5,214h256.2L818.6,0.4z"></path>
					<path class="st8" d="M818.6,0.4l128.1-213.5H690.5L818.6,0.4z"></path>
					<path class="st11" d="M1074.8,0.4L946.7,214h256.2L1074.8,0.4z"></path>
					<path class="st10" d="M1074.8,0.4l128.1-213.5H946.7L1074.8,0.4z"></path>
					<path class="st9" d="M49.9,427.5L-78.2,641H178L49.9,427.5z"></path>
					<path class="st5" d="M49.9,427.5L178,214H-78.2L49.9,427.5z"></path>
					<path class="st5" d="M306.2,427.5L178,641h256.2L306.2,427.5z"></path>
					<path class="st10" d="M306.2,427.5L434.3,214H178L306.2,427.5z"></path>
					<path class="st8" d="M562.4,427.5L690.5,214H434.3L562.4,427.5z"></path>
					<path class="st12" d="M818.6,427.5L690.5,641h256.2L818.6,427.5z"></path>
					<path class="st11" d="M1074.8,427.5L946.7,641h256.2L1074.8,427.5z"></path>
					<path class="st12" d="M1074.8,427.5L1202.9,214H946.7L1074.8,427.5z"></path>
					<path class="st5" d="M49.9,854.5L178,641H-78.2L49.9,854.5z"></path>
					<path class="st11" d="M306.2,854.5L434.3,641H178L306.2,854.5z"></path>
					<path class="st9" d="M562.4,854.5L690.5,641H434.3L562.4,854.5z"></path>
					<path class="st11" d="M818.6,854.5L946.7,641H690.5L818.6,854.5z"></path>
					<path class="st12" d="M1074.8,854.5L1202.9,641H946.7L1074.8,854.5z"></path>
				</g>
			</g>
		</g>
	</g>
	<rect class="st13" width="1120" height="641"></rect>
</g>
</svg>
