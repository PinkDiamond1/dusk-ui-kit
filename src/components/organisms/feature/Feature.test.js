import React from 'react'
import renderer from 'react-test-renderer'

import Feature from './Feature'

it('test to see if the Feature renders correctly', () => {
  const tree = renderer.create(<Feature>Smoke Test</Feature>).toJSON()
  expect(tree).toMatchSnapshot()
})
