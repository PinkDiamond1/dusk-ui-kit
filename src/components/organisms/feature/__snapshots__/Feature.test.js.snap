// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if the Feature renders correctly 1`] = `
<div
  className="__duk-feature  css-sbmh8s-Feature jumbotron"
>
  Smoke Test
</div>
`;
